library/options.inc.php

```diff
@@ -2036,7 +2036,7 @@ function generate_display_field($frow, $currvalue)
         }
         // If match is not found in main and backup lists, return the key with exclamation mark
         if ($s == '') {
-            $s = nl2br(htmlspecialchars($currvalue, ENT_NOQUOTES)).
+            $s = nl2br(xlt($currvalue)).
```

Comment:
Isn't this only showing if there is an error? (ie. the entry is missing)

Reply:
Right. It also prints the old value in the case the row was removed, but it is not translated..

Reply:
![image](https://user-images.githubusercontent.com/17809866/48183784-93137600-e338-11e8-9bdb-fc0c5c172b1f.png)

![image](https://user-images.githubusercontent.com/17809866/48183801-a4f51900-e338-11e8-901a-c93ef48b87eb.png)

Reply:
do the following instead:
```
xl_list_label
$s = nl2br(text(xl_list_label($currvalue))).
```

Reply:
To explain above xl_list_label relies on a option in globals to translate lists (it is on by default but gives folks option to turn off): https://www.open-emr.org/wiki/index.php/Administration_Globals#Translate_Lists

Reply:
fixed